src/prefect/client/orchestration.py

```diff
@@ -3012,11 +3012,11 @@ async def read_worker_metadata(self) -> Dict[str, Any]:
         return response.json()
 
     async def increment_concurrency_slots(
-        self, names: List[str], slots: int, mode: str
+        self, names: List[str], slots: int, mode: str, active: Optional[bool]
     ) -> httpx.Response:
         return await self._client.post(
             "/v2/concurrency_limits/increment",
-            json={"names": names, "slots": slots, "mode": mode},
+            json={"names": names, "slots": slots, "mode": mode, "active": active},
```

Comment:
I worry that this could be confusing for someone who has an existing concurrency limit that's inactive. I think we should make this argument obviously related to the _creation_ of the limit, like `create_as_active` or something.

Reply:
Totally agree. I hacked my way through here and thought the same thing.